src/components/Map/Map/Map.tsx

```diff
@@ -56,10 +61,13 @@ interface Props extends Omit<GoogleMap, 'onLoad' | 'mapContainerStyle' | 'option
 const Map = React.forwardRef<GoogleMap, Props>((props, ref) => {
     const { defaultArea, markers = [], mapContainerStyle, defaultHighlight, ...rest } = props;
     const mapRef = ref || React.createRef<GoogleMap>();
-    const [highlightFeatures, setHighlightFeatures] = React.useState<
+    const [defaultHighligh, setHighlightFeatures] = React.useState<
```

Comment:
maybe can change the name of the set function as well to match

Reply:
Fixed. Also renamed variable to defaultAreaHighlight to avoid confusion with defaultHighlight prop.